supabase/migrations/11_stats.sql

```diff
@@ -22,7 +22,7 @@ create table catalog_stats (
     failures            integer      not null default 0,
     ts                  timestamptz  not null,
     flow_document       json         not null
-);
+) partition by list (substring(catalog_name for position('/' in catalog_name)));
```

Comment:
We never dropped this table or its partitioning, so this migration is reverted back to reflect that.

New tenants won't have stats partitions created for them though, and new tenants won't have stats materializations created for them.